compiler+runtime/src/cpp/jank/runtime/obj/persistent_string.cpp

```diff
@@ -62,6 +63,41 @@ namespace jank::runtime::obj
     return data.compare(s.data);
   }
 
+  object_ptr persistent_string::get(object_ptr const key) const
+  {
+    return get(key, nil::nil_const());
+  }
+
+  object_ptr persistent_string::get(object_ptr const key, object_ptr const fallback) const
+  {
+    if(key->type == object_type::integer)
+    {
+      auto const i(static_cast<size_t>(expect_object<integer>(key)->data));
```

Comment:
Converting a signed value to an unsigned value will result in underflows. We'll want to handle the `< 0` case before casting, I think.

Makes sense to have a test case for negative values as well.